templates/labelprovider/contribution.ts

```diff
@@ -0,0 +1,33 @@
+import { MaybePromise } from "@theia/core";
```

Comment:
The other file has the header
```
/**
 * Generated using theia-extension-generator
 */
```
while this file has not. Although this is consistent to the other examples, this looks like an oversight to me ;) We can fix that in a follow up task.